sdcm/nemesis.py

```diff
@@ -3433,13 +3435,8 @@ def decommission_post_action():
             retry=0,
         )
 
-        terminate_patterns = ["DECOMMISSIONING: unbootstrap starts",
-                              "DECOMMISSIONING: unbootstrap done",
-                              "becoming a group 0 non-voter",
-                              "became a group 0 non-voter",
-                              "leaving token ring",
-                              "left token ring",
-                              "Finished token ring movement"]
+        terminate_patterns = ["DECOMMISSIONING: unbootstrap starts"] + \
```

Comment:
Why the `DECOMMISSIONING: unbootstrap starts` is not defined in the `get_log_patterns_for_decommission_abort` directly?

Reply:
moved messages to raft methods